pkg/kubectl/cmd/util/printing.go

```diff
@@ -52,7 +52,7 @@ func AddOutputFlags(cmd *cobra.Command) {
 
 // AddNoHeadersFlags adds no-headers flags to a command.
 func AddNoHeadersFlags(cmd *cobra.Command) {
-	cmd.Flags().Bool("no-headers", false, "When using the default or custom-column output format, don't print headers.")
+	cmd.Flags().Bool("no-headers", false, "When using the default or custom-column output format, print headers.")
```

Comment:
@tanshanshan thanks for the PR. However, the flag description isn't wrong. It describes the flag `--no-headers`, not `--no-headers=false` (here, `false` is the default value). 

If you feel the description is confusing, I suggest you update it to mention what the default behavior is.